core/src/main/resources/filodb-defaults.conf

```diff
@@ -465,6 +465,9 @@ filodb {
     # If all of the index-faceting-enabled-* properties are false, faceting is fully disabled.
     # Disable if performance cost of faceting all labels is too high
     index-faceting-enabled-for-all-labels = true
+
+    # Lucene IndexReader refresh frequency.
+    reader-refresh-interval = 6 hours
```

Comment:
6 hours default IMO is too high. Refresh in Lucene is not as expensive operation like commit and I think ``purgeExpiredPartitions`` is a good place to perform it. [ElasticSearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html#index-refresh-interval-setting) for example performs this refresh each second by default on its underlying index.  Our existing ``refreshReadersBlocking`` is not ideal as it will block all threads performing the refresh waiting for the refresh to finish. Instead we should use ``maybeRefresh`` in ``SearcherManager`` which will not block other threads while one of them performs the refresh.  We can also run performance benchmarks to assert our assumptions than have a pessimistic refresh interval. @vishramachandran  what's your view?